myscript.js

```diff
@@ -0,0 +1,51 @@
+// output function - BEGIN
+var outputArr = [];
+var outputLine = 0;
+var outputF = function (outputItem) {
+    outputArr [outputLine]=(outputItem);
+    outputLine = outputLine + 1;
+    $("#outputF").text(outputArr.join(" - "));
```

Comment:
you project does not have jquery installed, please add it to the project (as simple file)